mapit_gb/management/commands/mapit_UK_show_missing_codes.py

```diff
@@ -1,9 +1,24 @@
 from django.core.management.base import NoArgsCommand
 from mapit.models import Area, Generation
+from optparse import make_option
 
 
 class Command(NoArgsCommand):
     help = "Prints results of queries to check we aren't missing SNAC and GSS codes"
+    option_list = NoArgsCommand.option_list + (
+        make_option(
+            "--code_types", dest="code_types",
+            help="The list of code types to search for",
+            default="ons,gss"),
```

Comment:
Why not include `govuk_slug` in the default list of code types to check for? They're as essential to us in some contexts as the other two, so we really want to know if any are missing 🔍